<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monstro's Bestiary</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="entitesGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>9</y>
      <width>491</width>
      <height>661</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Entity</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="entitiesGridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>7</number>
      </property>
      <item row="2" column="4">
       <widget class="QLabel" name="portraitLabel">
        <property name="text">
         <string>Portrait ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="portraitBox">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="hpBaseLabel">
        <property name="text">
         <string>HP (base)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="hpBaseBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="hpStageLabel">
        <property name="text">
         <string>HP (stage)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="hpStageBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="frictionLabel">
        <property name="text">
         <string>Friction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="frictionBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="shieldLabel">
        <property name="text">
         <string>Shield</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="shieldBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="shadowSizeLabel">
        <property name="text">
         <string>Shadow Size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="shadowSizeBox">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="collisionGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>491</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Collision</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>468</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="collisionGroupLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="collisionRadiusLabel">
         <property name="text">
          <string>Collision Radius</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="collisionRadiusBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="collisionRadiusYMultyLabel">
         <property name="text">
          <string>CR Y Multi</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="collisionRadiusXMultiBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="collisionRadiusYMultiBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="gridCollisionLabel">
         <property name="text">
          <string>Grid Collison</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="gridCollisionInput"/>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="collisionDamageBox">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="collisionMassBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="collisionDamageLabel">
         <property name="text">
          <string>Collision Damage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="collisionMassLabel">
         <property name="text">
          <string>Collision Mass</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="collisionRadiusXMultyLabel">
         <property name="text">
          <string>CR X Multi</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="collisionIntervalLabel">
         <property name="text">
          <string>Collision Interval</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="gridCollLabel">
         <property name="text">
          <string>Gridcoll Points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="collisionIntervalBox">
         <property name="maximum">
          <number>1000000000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="gridCollPointsBox">
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="miscGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>491</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Misc</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>374</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="miscGroupLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="bossIDLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Boss ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="bossCheckBox">
         <property name="text">
          <string>Boss</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="bossIDBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="shutDoorsCheckBox">
         <property name="text">
          <string>Shut Doors</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="championCheckBox">
         <property name="text">
          <string>Champion</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="rerollCheckBox">
         <property name="text">
          <string>Reroll</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="xmlText">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>556</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="entitiesListBox" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QListWidget" name="entitiesList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>591</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Fixed</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="addEntityButton">
        <property name="text">
         <string>Add Entity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="removeEntityButton">
        <property name="text">
         <string>Remove Entity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
